istioctl/pkg/authz/listener.go

```diff
@@ -125,7 +125,15 @@ func ParseListener(listener *v2.Listener) *ParsedListener {
 	}
 
 	for _, fc := range listener.FilterChains {
-		parsedFC := &filterChain{tlsContext: fc.TlsContext}
+		tlsContext := &auth.DownstreamTlsContext{}
+		if fc.TransportSocket != nil && fc.TransportSocket.Name == "tls" {
```

Comment:
Are you sure it is "tls" - I thought we should configure it to `envoy.transport_sockets.tls` if we want to use the TLS transport socket?

Reply:
good catch, it was tls before, then got changed so I'll need to update this